dbfit-java/informix/src/main/java/dbfit/environment/InformixFunctionStatementExecution.java

```diff
@@ -0,0 +1,55 @@
+package dbfit.environment;
+
+import java.sql.*;
+
+import dbfit.fixture.StatementExecution;
+
+public class InformixFunctionStatementExecution extends StatementExecution {
+    private ResultSet rs;
+
+    public InformixFunctionStatementExecution(PreparedStatement statement, boolean clearParameters) {
+        super(statement, clearParameters);
+    }
+
+    @Override
+    public void run() throws SQLException {
+        rs = statement.executeQuery();
```

Comment:
What if we save the return value here and close the result set immediately after that?